### ISSUE TITLE
More style variations for the Theme Styles

### ISSUE THREAD

**Issue by mghenciu** [Codeinwp/neve-fse#49]
### What problem does this address?

I think we can do a better job in terms of colors, since right now it looks like the Palettes are not even different.

Here are some example of Neve FSE vs Raft vs Fork
![Screenshot 2023-06-21 at 14 03 31](https://github.com/Codeinwp/neve-fse/assets/52494172/00853254-ff56-40f0-bd56-23ad24ae067e)

___
![Screenshot 2023-06-21 at 14 03 57](https://github.com/Codeinwp/neve-fse/assets/52494172/289ec384-39d4-486b-91cd-953c7a70e9e0)

___

![Screenshot 2023-06-21 at 14 04 11](https://github.com/Codeinwp/neve-fse/assets/52494172/07b81f1a-764c-494b-a12a-f8300c62943e)

### What is your proposed solution?

I think we can customize the backgrounds a bit, in order to make them looks more distinct compared one to another.

**Comment by HardeepAsrani** [comment]
@JohnPixle How will it look with the different sets of variants that we have? Will it look better than it currently does overall?

I also see an issue with the Header background, it looks nice in the default but if you change the variants it does look a bit out of place. Let me know what you think.

**Comment by HardeepAsrani** [comment]
@JohnPixle Thanks, it makes more sense now.

**Comment by preda-bogdan** [comment]
Yes, I believe they use the same mechanism to display the preview there. The preview should be the same.
